packages/babel-compat-data/data/plugin-bugfixes.json

```diff
@@ -108,7 +108,6 @@
     "firefox": "2",
     "node": "6",
     "samsung": "5",
-    "rhino": "1.7.13",
```

Comment:
🤔Is Rhino affected by this Safari bug? I would expect this should not be deleted.

Reply:
I'll look into it, thanks.

Reply:
Hm, it doesn't appear to have the bug in either 1.7.13 or 1.7.14. Not sure why this happened.

Reply:
It looks like there are many `destructuring, parameters` tests marked as not passing in `compat-table`.

(Btw, I have to click on "show obsolete platforms" to see Rhino, you might want to change that)

Reply:
Hm, does that mean this one just has to be included as long as rhino is still failing any of those tests? I spent some time looking through this code but still don't really understand it.

Reply:
Oh well, the whole destructuring and parameters plugins will be included anyway (https://github.com/phulin/babel/blob/f5beb5ad5982d60d922d1756d9f274f92bcd6239/packages/babel-compat-data/data/plugins.json#L363, https://github.com/phulin/babel/blob/f5beb5ad5982d60d922d1756d9f274f92bcd6239/packages/babel-compat-data/data/plugins.json#L108), so this bugfix plugin is ignored because not needed.

It's strange that Rhino `1.7.13` was marked as passing this test in our data, but it doesn't matter :slightly_smiling_face: